### PR TITLE
chore(RHIF-219): Update /beta to /preview

### PR DESCRIPTION
**Pull request by johnsonm325**
To test, you can run via /beta or /preview. We previously had issues with the local app being served. To ensure that it is properly being served, run the app via beta and make a change in the code or add a console log and ensure that you're getting local changes.

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Review by leSamo — APPROVED** [review]

File: src/SmartComponents/DriftPage/__tests__/__snapshots__/DriftPage.tests.js.snap [hunk 1]
```diff
@@ -7345,14 +7367,15 @@ exports[`ConnectedDriftPage should render with error alert 1`] = `
                                                             </DropdownItem>,
                                                           ]
                                                         }
-                                                        isFlipEnabled={false}
+                                                        isFlipEnabled={true}
```
Comment (gkarat) [inline src/SmartComponents/DriftPage/__tests__/__snapshots__/DriftPage.tests.js.snap:7370]: Is it really supposed to change here?

File: src/SmartComponents/DriftPage/DriftTable/ReferenceSelector/__tests__/__snapshots__/ReferenceSelector.tests.js.snap [hunk 2]
```diff
@@ -109,6 +109,7 @@ exports[`ReferenceSelector should render mount correctly with isReference false
           "top-start": "pf-m-top-left",
         }
       }
+      removeFindDomNode={false}
```
Comment (gkarat) [inline src/SmartComponents/DriftPage/DriftTable/ReferenceSelector/__tests__/__snapshots__/ReferenceSelector.tests.js.snap:112]: Very suspicious tbh

**Review by gkarat — APPROVED** [review]
LGTM, but not sure about the snapshots. is it normal practice for drift to update them all with each PR? 🤔

File: src/SmartComponents/DriftPage/__tests__/__snapshots__/DriftPage.tests.js.snap (hunk 1 above)
Comment (johnsonm325) [reply src/SmartComponents/DriftPage/__tests__/__snapshots__/DriftPage.tests.js.snap:7370]: Patternfly had to be updated in order to update to frontend-components-utilities v 3.5.0. This is probably where all of this is coming from.

**Comment by Fewwy** [comment]
Other than that it works as expected, LGTM

**Review by Fewwy — APPROVED** [review]

**Pull request merged**